--- a/include/command.h	2019-10-08 00:14:02.000000000 +0300
+++ b/include/command.h	2026-03-12 22:16:12.283620188 +0200
@@ -177,7 +177,7 @@
  *			number of ticks the command took to complete.
  * @return 0 if the command succeeded, 1 if it failed
  */
-int cmd_process(int flag, int argc, char * const argv[],
+enum command_ret_t cmd_process(int flag, int argc, char * const argv[],
 			       int *repeatable, unsigned long *ticks);
 
 void fixup_cmdtable(cmd_tbl_t *cmdtp, int size);
