--- a/arch/arm/mach-sunxi/rsb.c	2019-10-08 00:14:02.000000000 +0300
+++ b/arch/arm/mach-sunxi/rsb.c	2026-03-12 22:17:49.594328406 +0200
@@ -19,7 +19,7 @@
 
 static void rsb_cfg_io(void)
 {
-#ifdef CONFIG_MACH_SUN8I
+#if defined(CONFIG_MACH_SUN8I) || defined(CONFIG_MACH_SUN50I)
 	sunxi_gpio_set_cfgpin(SUNXI_GPL(0), SUN8I_GPL_R_RSB);
 	sunxi_gpio_set_cfgpin(SUNXI_GPL(1), SUN8I_GPL_R_RSB);
 	sunxi_gpio_set_pull(SUNXI_GPL(0), 1);
